cvat/requirements/base.txt

```diff
@@ -33,3 +33,4 @@ djangorestframework==3.9.1
 Pygments==2.3.1
 drf-yasg==1.15.0
 Shapely==1.6.4.post2
+pascal_voc_writer==0.1.4
```

Comment:
Maybe it would be better, to install the dependency just in script like listed [here](https://stackoverflow.com/questions/12332975/installing-python-module-within-code) for example. 
It can be a sample for a user who wants to install another module to support a custom format. 
Also maybe we can provide some API to a module installation.
Just idea.